safe_transaction_service/safe_messages/tests/factories.py

```diff
@@ -33,4 +33,4 @@ class Params:
             encode_defunct(text=o.safe_message.message)
         )["signature"].hex()
     )
-    signature_type = 3
+    signature_type = 2
```

Comment:
Why was this changed?

Reply:
Signature is an EOA and not a ETH_SIGN

Reply:
But this is a factory? Or did you mean to change the default?

Reply:
No, is just a factory 😉